setup_yakut.sh

```diff
@@ -0,0 +1,6 @@
+export YAKUT_COMPILE_OUTPUT=.yakut
+export YAKUT_PATH="$YAKUT_COMPILE_OUTPUT"
```

Comment:
So, nowadays, the compile output and path are not needed (the compile command will be removed soon if not yet); instead, you simply specify where your DSDL sources are via `CYPHAL_PATH`. Isn't it _neat_? No more compiling!